src/main/scala/com/qubole/sparklens/QuboleJobListener.scala

```diff
@@ -155,6 +155,7 @@ class QuboleJobListener(sparkConf: SparkConf)  extends SparkListener {
       hostMap,
       executorMap,
       jobMap,
+      jobSQLExecIDMap,
```

Comment:
Can you please check if after this change, we can still parse sparklens.json files generated by older versions?